### ISSUE TITLE
Missing localization case for `INSTANT_YIELD_TYPE_COMBAT_EXPERIENCE` in `CvPlayer::doInstantYield()`

### ISSUE THREAD

**Issue by IanE9** [LoneGazebo/Community-Patch-DLL#10153]
The switch statement beginning on [line 28670 of CvPlayer.cpp](https://github.com/LoneGazebo/Community-Patch-DLL/blob/08b506b7c1174ba9225ea5be4f483b434bddeae6/CvGameCoreDLL_Expansion2/CvPlayer.cpp#L28670) covers all cases **except** `INSTANT_YIELD_TYPE_COMBAT_EXPERIENCE`.

**Comment by azum4roll** [comment]
It's meant to not have any notifications sent, but I guess we need to set the suppress flag somewhere and add an empty case for the new instant yield type.

**Comment by IanE9** [comment]
> It's meant to not have any notifications sent, but I guess we need to set the suppress flag somewhere and add an empty case for the new instant yield type.

Thanks for the info. I've added a case with #10154 to just return and documented why. That said, suppressing these sorts of warnings is almost never the right thing to do because they help find bugs, and I'm fairly sure in this scenario it's no different because the lack of a case meant that the switch would just fallen through to adding a notification with an empty message.

**Comment by azum4roll** [comment]
> > It's meant to not have any notifications sent, but I guess we need to set the suppress flag somewhere and add an empty case for the new instant yield type.
> 
> Thanks for the info. I've added a case with #10154 to just return and documented why. That said, suppressing these sorts of warnings is almost never the right thing to do because they help find bugs, and I'm fairly sure in this scenario it's no different because the lack of a case meant that the switch would just fallen through to adding a notification with an empty message.

No, I meant suppressing the notification, not the Clang warning.
